lib/index.js

```diff
@@ -9,6 +9,20 @@ if (typeof Promise === "undefined") {
   require("es6-promise").polyfill();
 }
 
+if (typeof Buffer === "undefined") {
+  const g = global || window;
+  g.Buffer = require("buffer").Buffer;
```

Comment:
@linuxwolf , I've noticed that you don't use `const` variable declaration anywhere in the project. For consistency, I can change these lines to either the following:
```
var g = global || window;
g.Buffer = require("buffer").Buffer;
```
OR
```
(global || window).Buffer = require("buffer").Buffer;
```

Reply:
Yeah, that's because this project started quite a few years ago, before `const` and `let` were as universally supported as they are now.

Thanks for sticking to consistency.

Reply:
I'm personally fine with either.

Reply:
I've updated to 2nd option for consistency. Marking this as resolved.